case/cheapino-top-left.scad

```diff
@@ -11,7 +11,7 @@ translate([0,0,9.1])
 rotate([0,180,0])
 difference() {
   case();
-  linear_extrude(4.1) offset(delta=0.15) base();
+  linear_extrude(4.1) offset(delta=0.45) base();
```

Comment:
It's worth noting that this was a little large. I think probably 0.35 would be sufficient, but 0.45 definitely produces a case that fits together easily.